qiskit_research/utils/convenience.py

```diff
@@ -38,14 +38,19 @@ def add_dynamical_decoupling(
     dd_str: str,
     scheduler: BaseScheduler = ALAPScheduleAnalysis,
     add_pulse_cals: bool = False,
+    urdd_pulse_num: int = 4,
 ) -> Union[QuantumCircuit, List[QuantumCircuit], List[List[QuantumCircuit]]]:
     """Add dynamical decoupling sequences and calibrations to circuits.
 
     Adds dynamical decoupling sequences and the calibrations necessary
     to run them on an IBM backend.
```

Comment:
It's finally time to document the arguments of this function.

Reply:
That's right, and it has been done.